plugin/bufferline.vim

```diff
@@ -86,7 +86,6 @@ command! -bang -complete=buffer -nargs=?
 let bufferline = extend({
 \ 'shadow': v:true,
 \ 'animation': v:true,
-\ 'icons': v:true,
```

Comment:
This is now a separate initialization step, and no default is registered for that slot until then.

However, if a user registers a value during this step then it will be retained during `bufferline#init_icons()`.